README.md

```diff
@@ -1013,16 +1013,17 @@ You can create, update, delete or load roles:
 name := "My Role"
 description := "Optional description to briefly explain what this role allows."
 permissionNames := []string{"My Updated Permission"},
-descopeClient.Management.Role().Create(context.Background(), name, description, permissionNames)
+tenantID := ""
```

Comment:
Worth adding here also a short sentence for what it can be used

Reply:
Done